requirements.txt

```diff
@@ -6,5 +6,5 @@ rlbot==1.*
 pip
 
 gevent
-eel
```

Comment:
Doesn't need to be part of this PR, just reminding myself: We should probably copy my branch of the eel source code right into this repo until it gets merged in and published.

If we accidentally publish an installer without my special branch, html, css and js will get cached for a long time on our users' browsers and it'll be a real mess.

Reply:
`https://github.com/tarehart/eel/archive/master.zip` should be used instead then